src/System.Devices.Gpio/GPIORef.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Devices.Gpio
```

Comment:
Are we going to vahe more devices? If not, I would put it in System.IO.Ports.Gpio (maybe?)